### PR TITLE
Add new prometheus versions and fix prom-label-proxy location

### PR DESCRIPTION
**Pull request by lilic**
Since github.com/prometheus-community/prom-label-proxy moved, we could remove the go mod override.

This also bumps the prometheus versions to latest two releases.

Sorry for the multiple things in one PR, just wanted to make sure we have these things before the release tomorrow.

cc @coreos/team-monitoring @vsliouniaev PTAL

### CODE REVIEW

**Comment by pgier** [comment]
@lilic I think this needs a vendor update `go mod vendor` and possibly `go mod tidy`

**Comment by lilic** [comment]
Ah yes, go mod tidy, thanks! PTAL, thanks!

File: go.mod [hunk 1]
```diff
@@ -34,8 +34,6 @@ require (
 )
 
 replace (
-	// Temporary until https://github.com/openshift/prom-label-proxy/pull/28 gets merged
-	github.com/openshift/prom-label-proxy => github.com/vsliouniaev/prom-label-proxy v0.0.0-20200518104441-4fd7fe13454f
 	// Prometheus 2.18.1
 	github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.0.0-20200507164740-ecee9c8abfd1
```
Comment (simonpasquier) [inline go.mod:38]: do we want to update that version too?

Comment (lilic) [reply go.mod:38]: Sure can do.

Comment (lilic) [reply go.mod]: Done @simonpasquier PTAL, thanks!

**Comment by benjaminhuo** [comment]
@lilic I think I still can use prometheus operator v0.38.1 to manage prometheus v2.19+ , right?

**Comment by metalmatze** [comment]
That's correct.

**Comment by benjaminhuo** [comment]
> That's correct.

Glad to know :)

**Pull request merged**